python/tank/commands/core_upgrade.py

```diff
@@ -181,6 +182,20 @@ def _run(self, log, core_version):
             log.error(msg)
             return_status = {"status": "update_blocked", "reason": msg}
 
+        elif status == TankCoreUpdater.UPDATE_BLOCKED_BY_CONFIG:
+            # The config is immutable so can't be updated.
+            engine = platform.current_engine()
+            descriptor_type = engine.sgtk.configuration_descriptor.get_dict().get(
```

Comment:
Errrr..... what happened here with this file?

Reply:
I'm not sure I understand? It looks OK to me?

Reply:
Ahhh sorry I see now, yeah this branch was based off another as yet unmerged branch, I just hadn't set the base correctly on the PR, sorry about that!